server/package.json

```diff
@@ -28,11 +28,11 @@
   },
   "dependencies": {
     "axios": "0.21.4",
-    "body-parser": "1.19.0",
+    "body-parser": "^1.20.1",
```

Comment:
I recommend pinning these to a specific version so that we're not having to deal with any potential incompatible updates

Reply:
True. I'll fix that when I get a moment.